src/AdapterPluginManagerFactory.php

```diff
@@ -41,6 +50,7 @@ public function __invoke(ContainerInterface $container, $name, ?array $options =
             return $pluginManager;
         }
 
+        /** @psalm-var ServiceManagerConfiguration $config */
```

Comment:
Shouldn't be necessary here?

Reply:
Forcing the type because psalm can't infer what `$container->get('config')['paginators']` is.